react-electron-poc/src/renderer/components/layout/Layout.tsx

```diff
@@ -100,134 +242,212 @@ const Layout = () => {
                 )
                 .catch((r) => console.log(r));
 
+            if (panelManager.current) panelManager.current.dispose();
             panelManager.current = new PanelManager(event);
-            /* const erb = panelFactory.addPanel('Erb', undefined, {
-                title: 'ERB',
-            }); */
-            /* const textPanel = panelFactory.addPanel(
-                'TextPanel',
-                {
-                    placeholderText: 'Loading zzz6 Psalm 119 USX',
-                    textPromise: getScripture('zzz6', 19, 119),
-                } as TextPanelProps,
-                {
-                    title: 'zzz6: Psalm 119 USX',
-                },
-            ); */
+
+            /* // Add the default tabs
+            if (panelManager.current.panelsInfo.size >= startingTabs) return;
```

Comment:
Is this comment still needed? Where does it end?

Reply:
Good call. I will take it out :) thanks!